packages/runner-ct/src/app/app.scss

```diff
@@ -50,7 +50,8 @@ main.app-ct {
     align-items: center;
     justify-content: center;
 
-    a {
+    a#menu-toggle {
```

Comment:
Without this menu button is 0px wide and cannot be found by `cy.get` - probably some global css present in the runner-ct but not in the component for whatever reason.